src/client/src/App.css

```diff
@@ -0,0 +1,269 @@
+.App {
```

Comment:
according to the instructions that CSS should have been splitted into multiple files according to the components and be using css modules
![image](https://user-images.githubusercontent.com/22417492/177046764-44c07342-ec4c-4b21-bab5-229ccbcccf90.png)